File: java/client/src/org/openqa/selenium/devtools/network/types/CertificateTransparencyCompliance.java
package org.openqa.selenium.devtools.network.types;

/**
 * Whether the request complied with Certificate Transparency policy
 */
public enum CertificateTransparencyCompliance {

  Unknown("unknown"),
  NotCompliant("not-compliant"),
  Compliant("compliant");

  private String compliance;

  CertificateTransparencyCompliance(String compliance) {
    this.compliance = compliance;
  }

  public String getCompliance() {
    return compliance;
  }
}

File: dotnet/src/webdriver/DevTools/AutoGenerated/Network/CertificateTransparencyCompliance.cs
namespace OpenQA.Selenium.DevTools.Network
{
    using Newtonsoft.Json;
    using Newtonsoft.Json.Converters;
    using System.Runtime.Serialization;

    /// <summary>
    /// Whether the request complied with Certificate Transparency policy.
    /// </summary>
    [JsonConverter(typeof(StringEnumConverter))]
    public enum CertificateTransparencyCompliance
    {
        [EnumMember(Value = "unknown")]
        Unknown,
        [EnumMember(Value = "not-compliant")]
        NotCompliant,
        [EnumMember(Value = "compliant")]
        Compliant,
    }
}